.asf.yaml

```diff
@@ -50,7 +50,6 @@ github:
         # See ./github/workflows/README.md for more documentation on this list.
         contexts:
            - Pulsar CI checks completed
-           - cpp-tests
```

Comment:
I'm afraid that you should remove this line alone without CI change or set up a dummy `cpp-tests` task since it's required on the master branch and doesn't respect changes in PR.

Otherwise, we may ask for help from the INFRA team to unrequire the check manually or even force merge with admin privilege.

Reply:
you're right. I removed the name from the job. in this way for this pull the job must pass. Once this is merged, we should be fine and we can add the job name again